packages/augur-ui/src/modules/portfolio/components/favorites/favorites.styles.less

```diff
@@ -57,4 +57,29 @@
       }
     }
   }
+}
+
+:root[theme='BETTING'],
+:root[theme='SPORTS'] {
+  .Watchlist {
+    div[class*='BoxHeader'] {
+      div[class*='SearchSort'] {
+        > div:first-of-type {
+          display: none;
+        }
+      }
+    }
+
+    > div:last-of-type {
+      min-height: @size-414;
+    }
+  }
+
+  .Watchlist__EmptyDisplay {
```

Comment:
I find that the height of Favorites component will be different when it's empty and not empty. I have spent some time on the current implementation. What do you think about it?

Reply:
It looks fine to me however can we change the class name? we spent time to remove classes with `__` or `-` in their names. Maybe just call this `EmptyDisplay` or `WatchlistEmptyDisplay`

Reply:
Thanks. Updated.